src/plugins/es_ui_shared/public/components/filter_list_button/filter_list_button.tsx

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
I'm not sure I see the benefits of having this is `es_ui_shared` right now since it is only being consumed by the Ingest Pipelines UI. Can it live in the ingest pipelines component?